buildlib/pr/distro.yml

```diff
@@ -155,10 +155,11 @@ jobs:
       - bash: |
           set -Eex
           ldd $(System.DefaultWorkingDirectory)/ucx_bin_$(Build.BuildId)/install/examples/hworld
-          UCX_LOG_LEVEL=info UCX_TLS=^sm LD_LIBRARY_PATH=$UCX_PR_LIB_PATH/ $(System.DefaultWorkingDirectory)/ucx_bin_$(Build.BuildId)/install/examples/hworld &
+          cmd_prefix="stdbuf -e0 -o0 timeout -s 9 300s"
```

Comment:
1. "UCX_LOG_LEVEL=info UCX_TLS=^sm LD_LIBRARY_PATH=$UCX_PR_LIB_PATH" could also be part of cmd_prefix
2. pls squash since it's a small PR

Reply:
`LD_LIBRARY_PATH=`  is different 
other will add